### PR TITLE
Add CODE-OF-CONDUCT.md and SECURITY.md

### PR DESCRIPTION
**Pull request by utam0k**
https://github.com/containers/youki/issues/62
I've decided to put the contact information for me in the profile. Or you can DM me on Twitter, which is written in the README.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Security and Disclosure Information Policy for the Youki Project
+
+The Buildah Project follows the [Security and Disclosure Information Policy](https://github.com/containers/common/blob/main/SECURITY.md) for the Containers Projects.
```
Comment (nimrodshn) [inline SECURITY.md:3]: @utam0k did you mean `Youki`?.

Comment (utam0k) [reply SECURITY.md]: Thasnk! I had missed the name change.

**Comment by nimrodshn** [comment]
LGTM :100:

**Comment by TomSweeneyRedHat** [comment]
LGTM
thanks for running this down @utam0k !

**Comment by utam0k** [comment]
@TomSweeneyRedHat Thanks for your review!

**Review by tsturzl — APPROVED** [review]
I think these are good things to establish with a growing contributor base. I'm going to approve this, but also put this out for consideration. The "Security and Disclosure Information Policy" directs security concerns to an email for podman. I'm not sure how or if  they'd forward these to us. Maybe we should consider having some kind of email that forwards to a group of primary collaborators for reporting security vulnerabilities. This is especially important if we follow the guide lines of following up on the issue in 3 working days.

It might be worth copying this notice out and updating is more specifically for our project?

**Comment by utam0k** [comment]
I also think Security Policy should be considered more. However, it would be better more later. For now, it's better than nothing, so let me set it up with this.
> I think these are good things to establish with a growing contributor base. I'm going to approve this, but also put this out for consideration. The "Security and Disclosure Information Policy" directs security concerns to an email for podman. I'm not sure how or if they'd forward these to us. Maybe we should consider having some kind of email that forwards to a group of primary collaborators for reporting security vulnerabilities. This is especially important if we follow the guide lines of following up on the issue in 3 working days.

**Comment by TomSweeneyRedHat** [comment]
FWIW,

The security email comes to me and a few other people.  We then forward it to the maintainers for the project to consider and fix if necessary.    Having an email listed in the SECURITY.md to send such emails to would be ideal, then we don't have to try and hunt you down.

**Pull request merged**